docs/contributing-tutorial.md

```diff
@@ -302,26 +302,23 @@ This will add a command which prints `Hello OSTree!` when `ostree hello-ostree`
         gboolean
         ostree_builtin_hello_ostree (int argc, char **argv, OstreeCommandInvocation *invocation, GCancellable *cancellable, GError **error)
         {
-          g_autoptr(GOptionContext) context = NULL;
           g_autoptr(OstreeRepo) repo = NULL;
-          gboolean ret = FALSE;
 
           // Creates new command context, ready to be parsed.
           // Input to g_option_context_new shows when running ostree <command> --help
-          context = g_option_context_new ("");
+          g_autoptr(GOptionContext) context = g_option_context_new ("");
 
           // Parses the command context according to the ostree CLI.
           if (!ostree_option_context_parse (context, options, &argc, &argv, invocation, &repo, cancellable, error))
-            goto out;
+            return FALSE;
 
           g_print("Hello OSTree!\n");
 
-          ret = TRUE;
-         out:
-          return ret;
+          return TRUE;
         }
 
     This defines the functionality for `hello-ostree`. Now we have to make sure the CLI can refer to the execution function, and that autotools knows to build this file.
+    Note: we should port to C99 style, for example https://github.com/ostreedev/ostree/pull/2651
```

Comment:
I'd either drop this sentence or reword into something like "Note: libostree codebase supports C99 features".